docs/api-reference/data-fetching/getStaticProps.md

```diff
@@ -0,0 +1,247 @@
+---
+description: Fetch data at build time with `getStaticProps`.
+---
+
+# `getStaticProps`
+
+<details>
+  <summary><b>Version History</b></summary>
+
+| Version   | Changes                                                                                                           |
+| --------- | ----------------------------------------------------------------------------------------------------------------- |
+| `v10.0.0` | `locale`, `locales`, `defaultLocale`, and `notFound` options added.                                               |
```

Comment:
Same comment for version history - non blocking

Reply:
https://github.com/vercel/next.js/issues/31953